src/autocomplete/autocomplete.scss

```diff
@@ -1,8 +1,40 @@
+// Stacked label (default)
+.autocomplete-label-stacked {
+  display: block;
+}
+
+// Inline label (non-default)
+.autocomplete-label-inline {
+  display: inline;
+
+  @media (max-width: $width-sm) {
+    display: block;
+  }
+}
+
+// Wrapper for the input and result elements to ensure alignment
+.autocomplete-body {
+  position: relative;
+}
+
+// Embedded icon
+.autocomplete-embedded-icon {
```

Comment:
I'd like to avoid absolute positioning things. This kind of design requires a bit more finagling and modification to `input`. I'll roughly spec out a suggestion here!

In the markup, wrap the icon and input in a `<div>`
```
<div class="form-control autocomplete-embedded-wrap">
<svg> 
<input class="form-control autocomplete-embedded-input" type="text" />
</div>
```

We'll need to strip the `input` of its styles for the most part, and modify the wrap to look like `form-control`

```
.autocomplete-embedded-wrap {
  // stylelint-disable-next-line primer/spacing
  padding: 4px 12px;
  display: inline-flex;
  align-items: center;
}

.autocomplete-embedded-input {
  border: none;
  box-shadow: none;
  padding: 0;
}
```

From here, you'll need to remove the `focus` state from `input` and apply to the wrap (kind of guessing here, might need some work)

```
.autocomplete-embedded-input:focus-within {
    border-color: var(--color-accent-emphasis);
    outline: none;
    box-shadow: var(--color-primer-shadow-focus);
}
```

Reply:
cc: @lindseywild <3

Reply:
This has been updated! The only thing we still need to address is the Firefox styling (you can see this in the screenshot in the below comment). I will ping again once it's ready!